posthog/user_scripts/latest_user_defined_function.xml

```diff
@@ -1,8 +1,8 @@
 <functions>
-    <!-- Version: v4. Generated at: 2024-12-10T05:06:27.315812+00:00
+    <!-- Version: v5. Generated at: 2025-03-19T21:09:12.111566+00:00
```

Comment:
are all the changes in this file intentional?

Reply:
i didn't touched this file directly. I ran the script that generates it (`python posthog/udf_versioner.py`)

as instructed on that file